src/components/ChatHeader.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import back from '../../public/images/back.png';
+import IU from '../../public/images/IU.png';
+import bell from '../../public/images/bell.png';
+import video from '../../public/images/video.png';
```

Comment:
```suggestion
import back from '/images/back.png';
import IU from '/images/IU.png';
import bell from '/images/bell.png';
import video from '/images/video.png';
```
`public` 폴더는 절대경로로 사용하실 수 있습니다! 이때에는 `public`을 생략하여 사용할 수 있습니다!